src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int firstSide;
+
+    public Square(int firstSide, Color color) {
+        this.firstSide = firstSide;
+        super.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * firstSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(

```

Comment:
In order to keep your code clean, consider creating a separate private method for the string used in the 'System.out.println' method.

Reply:
If I get you correctly it will generate one extra line. In the past I was asked about opposite action - removing all not necessarily lines and put them into return line. Correct me if I wrong, but it is completely arbitrary issue.